kusd/validator/utils.go

```diff
@@ -0,0 +1,51 @@
+package validator
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"math/big"
+
+	kowala "github.com/kowala-tech/kUSD"
+	"github.com/kowala-tech/kUSD/accounts/abi/bind"
+	"github.com/kowala-tech/kUSD/common"
+	"github.com/kowala-tech/kUSD/core/types"
+)
+
+func getTransactionOpts(contractBackend bind.ContractBackend, from common.Address, value *big.Int, chainID *big.Int) (*bind.TransactOpts, error) {
```

Comment:
this is a good place to test.
the func is loo large, you could split the anonymous func and test both in isolation